controls/radpivotgrid/features/queryabledataprovider/queryabledatetimegroupdescription.md

```diff
@@ -0,0 +1,256 @@
+---
+title: QueryableDateTimeGroupDescription
+page_title: QueryableDateTimeGroupDescription
+description: This article will describe the QueryableDateTimeGroupDescription class.
+slug: radpivotgrid-features-queryabledatetimegroupdescription
+tags: queryabledatetimegroupdescription
+published: True
+position: 9
+---
+
+# QueryableDateTimeGroupDescription
+
+In this article you will learn how to use a QueryableDateTimeGroupDescription with your DateTime and DateTimeOffset data.
+
+## QueryableDateTimeGroupDescription
+
+__QueryableDateTimeGroupDescription__ is used when you want to group your data by a property of type *DateTime* or *DateTimeOffset*. It has the following properties:      
+
+* __PropertyName__ - defines the name of the property by which the data will be grouped. It is mandatory to set a value.            
```

Comment:
Please use ":" instead of "-" for the list items. For example:

- PropertyName: Defines the name of the property...

This is the convention according to the Style Guide.